.github/workflows/deployStg.yml

```diff
@@ -108,14 +108,14 @@ jobs:
           client-tarball: client.tgz
           deploy-env: ${{env.DEPLOY_ENV}}
   verify-release:
-    runs-on: ubuntu-latest
     needs: [deploy]
-    defaults:
-      run:
-        working-directory: ./ops
+    if: "!contains(github.event.commits[0].message, '[skip e2e-dev]')"
+    runs-on: ubuntu-latest
+    env:
+      E2E_SPEC_FILES: "cypress/integration/00*"
+      CYPRESS_BASE_URL: https://stg.simplereport.gov
     steps:
-      - uses: actions/checkout@v2
-      - name: Run health checks
-        uses: ./.github/actions/health-checks
-        with:
-          deploy-env: ${{env.DEPLOY_ENV}}
\ No newline at end of file
+      - name: "Checkout"
+        uses: actions/checkout@v2
+      - name: Run Cypress checks against the dev remote environment
```

Comment:
Why run e2e tests twice as part of the same merge? If I'm reading this right, the tests will run in `dev` and then again in `stg` (but still against `dev`)?

Reply:
This is only running a health check and not the full suite. Per the other comments, this will just be changed to the full run.